ClockDisplay.java

```diff
@@ -73,6 +73,42 @@ public String getTime()
         return displayString;
     }
     
+    /**
+     * Get a display-able time in 12-hour(HH:MM AM/PM) format
+     * @return a String displaing the current time
+     */
+    public String get24HourInternalDisplay() 
```

Comment:
In this branch it should just be the method that updates the displayString field